zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ObjectResource.kt

```diff
@@ -53,4 +54,10 @@ class ObjectResource(
     ): ResponseEntity<URI> {
         return ResponseEntity.ok(zaakObjectService.patchObjectFromManagementId(objectManagementId, objectId, jsonNode))
     }
+
+    @GetMapping
```

Comment:
Missing HttpSecurityConfigurer

Reply:
Also, why do we need this?

Reply:
The Adam implementation needs this to get a call from form.io to the valtimo backend

Reply:
Not sure what you mean here. This endpoint here has nothing to do with form.io

Reply:
i was asked to make this endpoint by the implementation team to help them with a need that they would be able to use this call directly from within a form.io form or bpmn process